src/main/kotlin/apply/application/EvaluationDtos.kt

```diff
@@ -53,3 +54,33 @@ data class EvaluationResponse(
     val beforeEvaluationTitle: String = "",
     val beforeEvaluationId: Long = 0L
 )
+
+data class GradeEvaluationData(
+    @field:NotNull
+    @field:Valid
+    var gradeEvaluationItems: List<GradeEvaluationItemData> = emptyList(),
+
+    @field:Size(max = 255)
+    var note: String = "",
+
+    @field:NotNull
+    var evaluationStatus: EvaluationStatus = EvaluationStatus.WAITING,
+
+    var title: String = "",
+
+    var description: String = ""
+)
+
+data class GradeEvaluationItemData(
+    @field:NotNull
+    @field:Min(0)
+    var score: Int = 0,
+
+    var id: Long = 0L,
```

Comment:
애너테이션이 없으면 붙일 수 있을 것 같아요.

```suggestion
    var score: Int = 0,
    var id: Long = 0L,
```

Reply:
이 클래스의 이름을 `EvaluationAnswerData`로 변경했습니다.
그런데 이 클래스에서 `id`는 `evaluationItemId`를 의미합니다. 그런데 바인딩 때문에 `id`로 필드명을 뒀는데 괜찮을까요?
`BindingFormLayout`을 상속받게하고 id 필드는 따로 주는게 나을까요?